plugins/field-colour/README.md

```diff
@@ -1,6 +1,6 @@
 # @blockly/field-colour [![Built on Blockly](https://tinyurl.com/built-on-blockly)](https://github.com/google/blockly)
 
-A [Blockly](https://www.npmjs.com/package/blockly) colour field.
+A [Blockly](https://www.npmjs.com/package/blockly) colour field and blocks that use it.
```

Comment:
Open question: how should I describe the plugin, given that it is no longer just the colour field?

Reply:
perhaps something like "Fields and blocks for choosing and combining colours"?